.chloggen/elasticsearchexporter-store-process-executable-name.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

nit: Isn't this an enhancement?
```suggestion
change_type: enhancement
```